### PR TITLE
docs: add the document of the calibration status

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Description

Add the document for [this PR](https://github.com/tier4/tier4_ad_api_adaptor/pull/43).

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/design/autoware-interfaces/prototyping/api/external/get/accel_brake_map_calibrator/data.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- Behavior: Service
+- DataType: tier4_external_api_msgs/srv/GetAccelBrakeMapCalibrationData
+
+## Description
+
+AccelBrakeMap のキャリブレーション結果を取得する。
+
+## Requirement
+
+AccelBrakeMap のキャリブレーション結果、及び、評価用のグラフ画像を取得できること。
```
Comment (yabuta) [inline docs/design/autoware-interfaces/prototyping/api/external/get/accel_brake_map_calibrator/data.md]: キャリブレーション結果ではなく適正候補値とかにした方が良さそうです

File: docs/design/autoware-interfaces/prototyping/api/external/get/calibration_status.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Behavior: Topic
+- DataType: tier4_external_api_msgs/msg/CalibrationStatusArray
+
+## Description
+
+自動キャリブレーションによるパラメーターの評価状態を取得する。
+
+## Requirement
+
+キャリブレーションパラメーターの評価結果が取得できること。
```
Comment (yabuta) [inline docs/design/autoware-interfaces/prototyping/api/external/get/calibration_status.md]: ここはパラメータは不要な気がします
キャリブレーションの評価結果が取得できること
で良さそうです

**Review by yabuta — CHANGES_REQUESTED** [review]

File: docs/design/autoware-interfaces/prototyping/api/external/get/accel_brake_map_calibrator/data.md (hunk 1 above)
Comment (isamu-takagi) [reply docs/design/autoware-interfaces/prototyping/api/external/get/accel_brake_map_calibrator/data.md]: Fixed https://github.com/tier4/autoware-documentation/pull/21/commits/0f8c7a1971e94d6a5ee4adad425d25e24be5b7af.

File: docs/design/autoware-interfaces/prototyping/api/external/get/calibration_status.md (hunk 2 above)
Comment (isamu-takagi) [reply docs/design/autoware-interfaces/prototyping/api/external/get/calibration_status.md]: Fixed https://github.com/tier4/autoware-documentation/pull/21/commits/0f8c7a1971e94d6a5ee4adad425d25e24be5b7af.

**Review by yabuta — APPROVED** [review]
LGTM

**Pull request merged**